app/src/pages/playground/PlaygroundExperimentRunDetailsDialog.tsx

```diff
@@ -0,0 +1,189 @@
+    <Dialog
+      title={`Experiment Run for Example: ${exampleId}`}
+      size="fullscreen"
+    >
+      <PanelGroup direction="vertical" autoSaveId="example-compare-panel-group">
+        <Panel defaultSize={33}>
+          <View overflow="auto" height="100%" padding="size-200">
+            <Flex direction="row" gap="size-200" flex="1 1 auto">
+              <View width="50%">
+                <Card
+                  title="Input"
+                  {...defaultCardProps}
+                  bodyStyle={{
+                    padding: 0,
+                    maxHeight: "300px",
+                    overflowY: "auto",
+                  }}
+                  extra={<CopyToClipboardButton text={JSON.stringify(input)} />}
+                >
+                  <JSONBlock value={JSON.stringify(input, null, 2)} />
+                </Card>
+              </View>
+              <View width="50%">
+                <Card
+                  title="Reference Output"
+                  {...defaultCardProps}
+                  extra={
+                    <CopyToClipboardButton
+                      text={JSON.stringify(referenceOutput)}
+                    />
+                  }
+                  bodyStyle={{
+                    padding: 0,
+                    maxHeight: "300px",
+                    overflowY: "auto",
+                  }}
+                >
+                  <JSONBlock value={JSON.stringify(referenceOutput, null, 2)} />
+                </Card>
+              </View>
+            </Flex>
+          </View>
+        </Panel>
+        <PanelResizeHandle css={resizeHandleCSS} />
+        <Panel defaultSize={67}>
+          <Flex direction="column" height="100%">
+            <View
+              paddingStart="size-200"
+              paddingEnd="size-200"
+              paddingTop="size-100"
+              paddingBottom="size-100"
+              borderBottomColor="dark"
+              borderBottomWidth="thin"
+              flex="none"
+            >
+              <Heading level={2}>Experiment Run Output</Heading>
+            </View>
+            <div
+              css={css`
+                overflow-y: auto;
+                height: 100%;
+                padding: var(--ac-global-dimension-static-size-200);
+              `}
+            >
+              <Flex direction="row">
+                <View flex>
+                  {run.error ? (
+                    <View padding="size-200">
+                      <RunError error={run.error} />
+                    </View>
+                  ) : (
+                    <JSONBlock value={JSON.stringify(run.output, null, 2)} />
+                  )}
+                </View>
+                <ViewSummaryAside width="size-3000">
+                  {run.startTime && run.endTime && (
+                    <RunLatency
+                      startTime={run.startTime}
+                      endTime={run.endTime}
+                    />
+                  )}
+                </ViewSummaryAside>
```

Comment:
I know you can't annotate runs yet but if you leave thatin here we get it for free later when we add it